app/assets/javascripts/waitForCIAPIFixedChanges.js

```diff
@@ -0,0 +1,36 @@
+import * as elementWatcher from './waitForEl'
+import * as statusObserver from './statusObserver'
+import * as dataLayerUpdater from './updateDatalayer'
+import {createDataLayerElement,reportEvent} from './addToDataLayer'
+import {availabilities} from './getAvailability'
+
+export function waitForCIAPIFixedChanges(w, d) {
+  w.addEventListener("load", function () {
+      waitForNuanceElement('#HMRC_CIAPI_Fixed_1',w,d);
+  });
+};
+
+function waitForNuanceElement(el,w,d) {
+  elementWatcher.waitForEl(el + ' div',
+     function () {
+      dataLayerUpdater.updateDataLayer(el,w,d);
+      statusObserver.observeStatus(el,w,d);
+    },
+    function() {
+      const assistantUsed = w.location.pathname.includes("webchat")
+        ? "webchat"
+        : "the digital assistant";
+
+      var the_element = d.querySelector(el);
+      the_element.textContent = "There's a problem with " + assistantUsed + ". Try again later.";
+
+      reportEvent(w,createDataLayerElement(availabilities.NuanceUnavailable, el))
+
+      var elements_to_hide = d.querySelectorAll(".hide-text-on-error");
+      for (var i = 0; i < elements_to_hide.length; i++) {
+        elements_to_hide[i].style.display = 'none';
+      }
+    }
+  );
+
```

Comment:
I am not sure this variable or loop are being used. I tried console logging them and switching the cookies off and I got an empty node list. I cannot find that class in either the skin, the frontend other than in the tests or in the inspect tool elements section. I can see it is also in the waitForChanges.js file but blocking the cookies did not prevent the Nuance chat from loading so I could not do an equivalent check on that one.

I have approved the PR as it does make me slightly uncomfortable removing this as I presume it was put in for a good reason and I am potentially not recreating the circumstances correctly.